apps/OboeTester/app/src/main/cpp/LatencyAnalyzer.h

```diff
@@ -478,12 +478,12 @@ class LoopbackProcessor {
 
     virtual int save(const char *fileName) {
         (void) fileName;
-        return AAUDIO_ERROR_UNIMPLEMENTED;
+        return -1;
     }
 
     virtual int load(const char *fileName) {
         (void) fileName;
-        return AAUDIO_ERROR_UNIMPLEMENTED;
+        return -1;
```

Comment:
Add `// AAUDIO_ERROR_UNIMPLEMENTED` to show what this "magic number" corresponds to.